### ISSUE TITLE
[CRITICAL] Plugin broken when upgrading Vagrant 2.4.1 -> 2.4.2

### ISSUE THREAD

**Issue by lprimak** [hashicorp/vagrant-vmware-desktop#131]
# CRITICAL - There is no way to restore vagrant VMware functionality while using Homebrew

### Vagrant version

Vagrant 2.4.2

### Vagrant VMware plugin version
Latest

### Vagrant VMware utility version
N/A

### Host operating system
MacOS

This is the operating system that you run locally.

### Guest operating system
Linux

This is the operating system you run in the virtual machine.

### Debug output

```
% vagrant plugin install vagrant-vmware-desktop
Installing the 'vagrant-vmware-desktop' plugin. This can take a few minutes...
Vagrant failed to properly resolve required dependencies. These
errors can commonly be caused by misconfigured plugin installations
or transient network issues. The reported error is:

conflicting dependencies logger (= 1.6.0) and logger (= 1.6.1)
  Activated logger-1.6.1
  which does not match conflicting dependency (= 1.6.0)

  Conflicting dependency chains:
    logger (= 1.6.1), 1.6.1 activated

  versus:
    logger (= 1.6.0)

  Gems matching logger (= 1.6.0):
    logger-1.6.0
```
### Steps to reproduce

1. `vagrant plugin install vagrant-vmware-desktop`

**Comment by silverskyvicto** [comment]
I faced the same problem.

**Comment by filoucrackeur** [comment]
same probem

**Comment by mickaelperrin** [comment]
Got the same issue, this error is also reported on vagrant project. A fix has already been commited.

A temporary fix is:

```
VAGRANT_DISABLE_STRICT_DEPENDENCY_ENFORCEMENT=1
```

see https://github.com/hashicorp/vagrant/issues/13527

**Comment by lprimak** [comment]
How would I know if the fix has been released? 2.4.3? Something else?
Thanks!

**Comment by brlin-tw** [comment]
> How would I know if the fix has been released? 2.4.3? Something else?

It should be in 2.4.3 as long as my patch isn't reverted afterwards.

**Comment by lprimak** [comment]
Verified fixed
